app/routes/newsletter/nota-updates.js

```diff
@@ -20,7 +20,7 @@ export default class NewsletterNotaUpdatesRoute extends Route {
   *pollModel() {
     while (true) {
       yield timeout(3 * 60000);
-      this.refresh();
```

Comment:
calling `router` in a route refreshes the parent route it seems.
just using local `this.refresh()` only reloads our own model.